include/zephyr/bluetooth/hci.h

```diff
@@ -534,6 +534,53 @@ struct bt_hci_rp_read_tx_power_level {
 	int8_t   tx_power_level;
 } __packed;
 
+#define BT_HCI_LE_TX_POWER_PHY_1M               0x01
+#define BT_HCI_LE_TX_POWER_PHY_2M               0x02
+#define BT_HCI_LE_TX_POWER_PHY_CODED_S8         0x03
+#define BT_HCI_LE_TX_POWER_PHY_CODED_S2         0x04
+#define BT_HCI_OP_LE_ENH_READ_TX_POWER_LEVEL    BT_OP(BT_OGF_LE, 0x0076)
+struct bt_hci_cp_le_read_tx_power_level {
+	uint16_t handle;
+	uint8_t  phy;
+} __packed;
+
+struct bt_hci_rp_le_read_tx_power_level {
+	uint8_t  status;
+	uint16_t handle;
+	uint8_t  phy;
+	int8_t   current_tx_power_level;
+	int8_t   max_tx_power_level;
+} __packed;
+
+#define BT_HCI_OP_LE_READ_REMOTE_TX_POWER_LEVEL	BT_OP(BT_OGF_LE, 0x0077)
+
+#define BT_HCI_OP_LE_SET_PATH_LOSS_REPORT_PARAM BT_OP(BT_OGF_LE, 0x0078)
+struct bt_hci_cp_le_set_path_loss_report_param {
+	uint16_t handle;
+	int8_t high_threshold;
```

Comment:
You've used the appropriate alignment of struct member names for `bt_hci_rp_le_read_tx_power_level` and `bt_hci_cp_le_set_path_loss_report_enable`, but here you're missing two spaces after `int8_t` (so `high_threshold` aligns with `handle` on the preceding line). Please try to be consistent with this :)